config/prow/plugins.yaml

```diff
@@ -201,3 +201,7 @@ external_plugins:
     events:
       - issue_comment
       - pull_request
+  - name: cherrypicker
```

Comment:
That's awesome!
Just a question: does this carry along the release notes and labels? ref https://github.com/gardener/gardener/pull/3690 https://github.com/gardener/gardener/pull/4053 
If not, we can also think about ways to add this later on.

Reply:
I'm not sure about the labels, but I afaik it is copying the release notes
When you look at this PR created by cherrypicker you will find the release notes of its parent PR `https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2959`

Reply:
Ok, let's try it out :)